extractData.go

```diff
@@ -0,0 +1,37 @@
+package main
+
+import (
+	"encoding/json"
+	"log"
+	"net/http"
+	"os"
+)
+
+func extractData[T Wind | Basic](w http.ResponseWriter, filenm string, params T) error {
```

Comment:
Prefer `fileName`?

Reply:
To be even more clear, it would be`filePath`?

Reply:
Yeah